src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -154,6 +156,16 @@ public IEnumerable<DataSource> ListAllDataSources()
         return _dataSourceNameToDataSource.AsEnumerable();
     }
 
+    public bool TryAddEntityPathNameToEntityName(string entityPathName, string entityName)
```

Comment:
I noticed these and a couple other methods come before the constructor in this class. Normally the order is contructors before methods, was there a reason in particular that these (and the others) come first?

Reply:
I think this was a styling thing missed in earlier pr's. I dont want to bloat this pr, will fix this styling issue in a follow up pr that addresses this style concern.